packages/mdc-layout-grid/_mixins.scss

```diff
@@ -48,6 +48,9 @@
 
   @supports (display: grid) {
     width: auto;
+    max-width: 100%;
+    max-height: 100%;
+    overflow: hidden;
```

Comment:
@yeelan0319 Is this clipping the correct behavior? Do we want the image to be contained or covering the parent?